app/controllers/reading_collections_controller.rb

```diff
@@ -1,7 +1,9 @@
 class ReadingCollectionsController < ApplicationController
   def show
-    @collection = ReadingCollection.find_by(slug: params[:slug]).to_json
-    @articles = ReadingCollection.find_by(slug: params[:slug]).articles.to_json
+    reading_collection = ReadingCollection.find_by(slug: params[:slug])
+
+    @collection = reading_collection.to_json
+    @articles = reading_collection.articles.includes([:taggings]).to_json
```

Comment:
Joel - the failing tests forced me to change this and include the `includes([:taggings])` in the query to get articles. I triple checked and it does not seem to be affecting the FE.

Reply:
Swell, I'll check  on my end to be sure, but probably good to have in there regardless!